WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -22,8 +22,8 @@ decisions = {
 				narrative_event = { id = guard.3100 }
 				create_character = {
 					random_traits = yes
-					religion = nicene
-					culture = greek
+					religion = ROOT
+					culture = ROOT
```

Comment:
Why not just make it so that only Christians can create it and have it always Greek?